test/data/SampleData.scala

```diff
@@ -67,6 +68,7 @@ object SampleData {
 
   val chargeBDetails = ChargeBDetails(4, chargeAmount1)
   val memberDetails: MemberDetails = MemberDetails("first", "last", "AB123456C")
+  val memberDetailsG: MemberDetailsG = MemberDetailsG("first", "last", LocalDate.now(), "AB123456C")
```

Comment:
Could this be given a more meaningful name?

Reply:
This follows the naming convention used in the backend

Reply:
OK